Sources/Tabs/MainCoordinator.swift

```diff
@@ -9,141 +9,47 @@ import UIKit
     private let walletAvailabilityService: WalletFeatureAvailabilityService
-    private let networkClient: NetworkClient
     
     private weak var loginCoordinator: LoginCoordinator?
     private weak var homeCoordinator: HomeCoordinator?
     private weak var walletCoordinator: WalletCoordinator?
     private weak var profileCoordinator: ProfileCoordinator?
     
-    init(windowManager: WindowManagement,
+    init(appWindow: UIWindow,
          root: UITabBarController,
          analyticsCenter: AnalyticsCentral,
          networkClient: NetworkClient,
          sessionManager: SessionManager,
-         updateService: AppInformationServicing = AppInformationService(),
          walletAvailabilityService: WalletFeatureAvailabilityService = WalletAvailabilityService()) {
-        self.windowManager = windowManager
+        self.appWindow = appWindow
         self.root = root
         self.analyticsCenter = analyticsCenter
         self.networkClient = networkClient
         self.sessionManager = sessionManager
-        self.updateService = updateService
         self.walletAvailabilityService = walletAvailabilityService
     }
     
     func start() {
-        addTabs()
-        windowManager.displayUnlockWindow(analyticsService: analyticsCenter.analyticsService) { [unowned self] in
-            evaluateRevisit()
-        }
-        evaluateRevisit()
         root.delegate = self
-        NotificationCenter.default
-            .addObserver(self,
-                         selector: #selector(startReauth),
-                         name: Notification.Name(.startReauth),
-                         object: nil)
-    }
-    
-    func evaluateRevisit() {
-        guard sessionManager.expiryDate != nil else {
-            fullLogin()
-            return
-        }
-
-        guard sessionManager.isSessionValid else {
-            fullLogin(loginError: TokenError.expired)
-            return
-        }
-
-        Task {
-            await MainActor.run {
-                do {
-                    try sessionManager.resumeSession()
-                    updateToken()
-                    windowManager.hideUnlockWindow()
-                } catch {
-                    switch error {
-                    case is JWTVerifierError,
-                        SecureStoreError.unableToRetrieveFromUserDefaults,
-                        SecureStoreError.cantInitialiseData,
-                        SecureStoreError.cantRetrieveKey:
-                        fullLogin(loginError: error)
-                    default:
-                        print("Token retrival error: \(error)")
-                    }
-                }
-            }
-        }
-    }
-    
-    @objc private func startReauth() {
-        fullLogin(loginError: TokenError.expired)
-    }
-    
-    private func fullLogin(loginError: Error? = nil) {
-        sessionManager.endCurrentSession()
-        showLogin(loginError)
-        windowManager.hideUnlockWindow()
+        addTabs()
```

Comment:
All calls to local auth have been removed from the MainCoordinator as this coordinator will be solely responsible for managing the tab bar functionality of the app.

Reply:
Can we maybe get this documented somewhere that this is what the main coordinator is for now? I feel like I'd forget this 😅

Reply:
Yep absolutely, we've talked about renaming this coordinator as it actually just manages the tabs now. Unfortunately "TabCoordinator" is the name of the protocol so it's been left as is but, yes, let's rename and leave a bit of DocC at the top of the class to make this obvious 🙂